pkg/webui/lib/host-from-url.js

```diff
@@ -20,7 +20,11 @@
  */
 export default url => {
   try {
-    return new URL(url).hostname
+    if (url.match(/^[a-zA-Z0-9]+:\/\/.*/)) {
+      return new URL(url).hostname
+    }
+
+    return new URL(`http://${url}`).hostname
   } catch (error) {
     return undefined
```

Comment:
```suggestion
    return url
```

I think that's more resilient.